src/style.css

```diff
@@ -0,0 +1,89 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  width: 100%;
+  min-width: 300px;
+  text-align: center;
+  line-height: 300px;
+  flex: 1 1 33.3%;
+  color: hsl(0, 0%, 100%);
+  font-size: inherit;
+  font-family: inherit;
+}
+
+.first-block {
+  background-color: hsl(0, 100%, 50%);
+  order: 1;
+}
+
+.second-block {
+  background-color: hsl(0, 100%, 40%);
+  order: 2;
+}
+
+.third-block {
+  background-color: hsl(0, 100%, 30%);
+  order: 3;
+}
```

Comment:
you do not need to set the order of the first three blocks, they are in the order you need by default